doc/conf.py

```diff
@@ -223,23 +230,87 @@
 # Register the theme as an extension to generate a sitemap.xml
 # extensions.append("guzzle_sphinx_theme")
 
+
+html_sidebars = {
+    '**' : [
+        'logo-text.html',
+        'searchbox.html',
+        'globaltoc.html',
+        # 'sourcelink.html'
+    ]
+}
+
+# Additional templates that should be rendered to pages, maps page names to
+# template names.
+#html_additional_pages = {}
+
+# If false, no module index is generated.
+#html_domain_indices = True
+
+# If false, no index is generated.
+#html_use_index = True
+
+# If true, the index is split into individual pages for each letter.
+#html_split_index = False
+
+# If true, links to the reST sources are added to the pages.
+#html_show_sourcelink = True
+
+# If true, "Created using Sphinx" is shown in the HTML footer. Default is True.
+#html_show_sphinx = True
+
+# If true, "(C) Copyright ..." is shown in the HTML footer. Default is True.
+#html_show_copyright = True
+
+# If true, an OpenSearch description file will be output, and all pages will
+# contain a <link> tag referring to it.  The value of this option must be the
+# base URL from which the finished HTML is served.
+#html_use_opensearch = ''
+
+# This is the file name suffix for HTML files (e.g., ".xhtml").
+#html_file_suffix = None
+
+# Language to be used for generating the HTML full-text search index.
+# Sphinx supports the following languages:
+#   'da', 'de', 'en', 'es', 'fi', 'fr', 'h', 'it', 'ja'
+#   'nl', 'no', 'pt', 'ro', 'r', 'sv', 'tr'
+#html_search_language = 'en'
+
+# A dictionary with options for the search language support, empty by default.
+# Now only 'ja' uses this config value
+#html_search_options = {'type': 'default'}
+
+# The name of a javascript file (relative to the configuration directory) that
+# implements a search results scorer. If empty, the default will be used.
+#html_search_scorer = 'scorer.js'
+
+# Output file base name for HTML help builder.
+htmlhelp_basename = 'PennyLaneQiskitdoc'
+
+# # -- Xanadu theme ---------------------------------------------------------
+html_theme = 'xanadu_theme'
+html_theme_path = ['.']
+
+# Register the theme as an extension to generate a sitemap.xml
+# extensions.append("guzzle_sphinx_theme")
+
 # xanadu theme options (see theme.conf for more information)
 html_theme_options = {
 
     # Set the path to a special layout to include for the homepage
     # "index_template": "special_index.html",
 
     # Set the name of the project to appear in the left sidebar.
-    #"project_nav_name": "PennyLane-qiskit",
+    "project_nav_name": "PennyLane-Qiskit",
```

Comment:
Would it be possible to add some sort of hyperlink that takes us back to the `About` page when clicked on? Each PL specific link takes us back to PL itself and was wondering if a PL-Qiskit specific link could also be added. (Not a must from my side, especially if it's something complicated to set.)

Reply:
Wait, the "About" link in the sidebar is a clickable toc entry...Is that what you mean?

Reply:
That's true, it is already clickable! The thought here was just to further make it as a landing page for the entire plugin documentation and create another link by clicking on the PL-Qiskit part as well. The reason for that would just be that it's a slightly larger clickable that might easily draw the attention of the visitor on the sidebar, should they want to go back to a basic landing page.

Though your point is completely valid, that would be a duplicate for being directed to the About page! (Therefore I'm totally fine with leaving it as is)

Reply:
Perfect. I know exactly what you mean, would be great to have some logo to click for "home"... I think that one is non-trivial though...